src/renderer/containers/Console/AddConnectionForm.tsx

```diff
@@ -187,7 +189,35 @@ export const AddConnectionForm: React.FC<AddConnectionFormProps> = ({ defaultVal
                         if (!val) {
                           return false;
                         }
-                        return validateIpAndPort(val);
+                        return isValidIpAddress(val);
+                      },
+                    }}
+                  />
+                  <Controller
+                    name="obsPort"
+                    control={control}
+                    defaultValue=""
+                    render={({ field, fieldState: { error } }) => (
+                      <TextField
+                        {...field}
+                        label="OBS Websocket Port"
+                        required={enableAutoSwitcher}
+                        error={Boolean(error)}
+                        helperText={error ? error.message : undefined}
+                        disabled={disabled}
+                      />
+                    )}
+                    rules={{
+                      required: enableAutoSwitcher,
+                      validate: (val) => {
+                        if (!enableAutoSwitcher) {
+                          return true;
+                        }
+                        if (!val) {
+                          return false;
+                        }
+                        const port = parseInt(val);
+                        return (port > 0 && port < 65535) || "Invalid Port";
```

Comment:
```suggestion
                        return val && isValidPort(val) || "Invalid Port";
```
Should we have this in a separate function?

```typescript
function isValidPort(value: string | number) {
  const port = typeof value === "string" ? parseInt(value) : value;
  return port > 0 && port < 65535);
}
```